src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/ReleaseFileServiceTests.cs

```diff
@@ -1171,6 +1172,125 @@ await using (var contentDbContext = InMemoryApplicationDbContext(contentDbContex
             MockUtils.VerifyAllMocks(blobStorageService);
         }
 
+        [Fact]
+        public async Task GetFile()
+        {
+            var releaseFile = new ReleaseFile
+            {
+                Release = new Release(),
+                Name = "Test PDF File",
+                File = new File
+                {
+                    RootPath = Guid.NewGuid(),
+                    Filename = "ancillary.pdf",
+                    Type = Ancillary
+                }
+            };
+
+            var contentDbContextId = Guid.NewGuid().ToString();
+            await using (var contentDbContext = InMemoryApplicationDbContext(contentDbContextId))
+            {
+                await contentDbContext.AddAsync(releaseFile);
+                await contentDbContext.SaveChangesAsync();
+            }
+
+            var blobStorageService = new Mock<IBlobStorageService>(MockBehavior.Strict);
+            var blob = new BlobInfo(
+                path: null,
+                size: "93",
```

Comment:
Doesn't really mean anything but I think due to the way it's set it will always have some sort of size suffix on it like "800 B" or "93 Kb"